cms/envs/test.py

```diff
@@ -341,22 +332,16 @@
 
 SERVICE_VARIANT = os.environ.get('SERVICE_VARIANT', None)
 
-# CONFIG_ROOT = path(os.environ.get('CONFIG_ROOT', ENV_ROOT))
-
-# CONFIG_PREFIX = SERVICE_VARIANT + "." if SERVICE_VARIANT else ""
-
-# with open(CONFIG_ROOT / CONFIG_PREFIX + "auth.json") as auth_file:
-#     AUTH_TOKENS = json.load(auth_file)
-
-AUTH_TOKENS = dict()
-
+##################### PhilU Settings #######################
 NODEBB_RETRY_DELAY = 60
 NODEBB_ENDPOINT = "http://local.philanthropyu.org:4567"
-# replace NODEBB_MASTER_TOKEN with value from your setup
-NODEBB_MASTER_TOKEN = AUTH_TOKENS.get("NODEBB_MASTER_TOKEN")
-MANDRILL_API_KEY = AUTH_TOKENS.get("MANDRILL_API_KEY")
-MAILCHIMP_API_KEY = AUTH_TOKENS.get("MAILCHIMP_API_KEY")
+NODEBB_MASTER_TOKEN = 'test-master-token-nodebb'
+MANDRILL_API_KEY = 'test_mandrill_api_key_part1-part2-part3'
+# TODO: Mock usages of mailchimp client in all unit tests and replace this with dummy key
+MAILCHIMP_API_KEY = '21bf1dc52e6ef64a6a3f573e4ea9a999-us12'
```

Comment:
is that fine to putt key directly in this file  in the case of test.py ?

Reply:
I won't say its fine but this is the key for development. And if you refer to TODO it states how we can get rid of it.

Reply:
got it, thanks :)

Reply:
If this is an actual key, then we should not keep it in code. Even it should not be in git history